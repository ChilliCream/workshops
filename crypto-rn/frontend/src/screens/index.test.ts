import * as Screens from './index';

describe('Screens', () => {
  it('should keep shape', () => {
    expect(Screens).toMatchSnapshot();
  });
});
