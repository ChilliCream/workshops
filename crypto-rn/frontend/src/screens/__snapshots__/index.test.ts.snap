// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens should keep shape 1`] = `
{
  "Home": [Function],
}
`;
